Tests/SonarScanner.MSBuild.PreProcessor.Test/SonarCloudWebServerTest.cs

```diff
@@ -49,51 +46,36 @@ public class SonarCloudWebServerTest
     private const string Organization = "org42";
 
     private readonly TimeSpan httpTimeout = TimeSpan.FromSeconds(42);
-    private readonly IDownloader downloader;
-    private readonly IJreCache jreCache;
-    private readonly Version version;
-    private readonly TestLogger logger;
-
-    private SonarCloudWebServer sut;
-
-    public SonarCloudWebServerTest()
-    {
-        downloader = Substitute.For<IDownloader>();
-        jreCache = Substitute.For<IJreCache>();
-        version = new Version("5.6");
-        logger = new TestLogger();
-    }
-
-    [TestInitialize]
-    public void Init() =>
-        sut = CreateServer();
-
-    [TestCleanup]
-    public void Cleanup() =>
-        sut?.Dispose();
+    private readonly Version version = new Version("5.6");
+    private readonly TestLogger logger = new TestLogger();
```

Comment:
I think we need to be careful with the logger. If it's reused between tests, it might have logs already when a test method is started.